python/py-typing/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-typing
+version             3.5.2.2
+categories-append   devel
+license             PSF
+platforms           darwin
+supported_archs     noarch
+
+maintainers         gmail.com:allan.que openmaintainer
+
+description         backport of the standard library typing module
+long_description    ${name} is a backport of the standard library typing module to Python \
+                    versions older than 3.5. It defines a standard notation for Python \
+                    function and variable type annotations. The notation can be used for \
+                    documenting code in a concise, standard format, and it has been designed \
+                    to also be used by static and runtime type checkers, static analyzers, \
+                    IDEs and other tools.
+
+homepage            http://pypi.python.org/pypi/typing/
+master_sites        pypi:t/typing
+
+distname            typing-${version}
+
+checksums           rmd160  8bd175ae415b89da4edc2d9ffc2dada1a2bb8a2a \
+                    sha256  2bce34292653af712963c877f3085250a336738e64f99048d1b8509bebc4772f
+
+python.versions     27 32 33 34
+
+if {${name} ne ${subport}} {
+    livecheck.type      none
+} else {
+    livecheck.type      regex
+    livecheck.url       https://pypi.python.org/pypi/typing/json
+    livecheck.regex     {typing-(\d+(?:\.\d+)*)}
```

Comment:
You should omit the `else` clause here; a PyPI-based livecheck will be added automatically.